github-pullrequest-plugin/src/main/java/com/github/kostyasha/github/integration/branch/GitHubBranchRepository.java

```diff
@@ -53,6 +53,14 @@ public GitHubBranchRepository(GHRepository remoteRepository) {
         return branches;
     }
 
+    public void addBranch(GitHubBranch branch) {
```

Comment:
why? why should i have to access the internal map to add/get a branch? it exposes internal implementation details.

also - as a delegate method, it makes unit testing easier.